test/bokken_web/controllers/organizer_json_test.exs

```diff
@@ -0,0 +1,36 @@
+defmodule Bokken.OrganizerJSONTest do
+  use Bokken.DataCase
+
+  import Bokken.Factory
+  alias BokkenWeb.OrganizerJSON
+
+  test "data" do
+    organizer = build(:organizer)
+    rendered_organizer = OrganizerJSON.data(organizer)
+
+    assert rendered_organizer == %{
+             id: organizer.id,
+             champion: organizer.champion,
+             first_name: organizer.first_name,
+             last_name: organizer.last_name
+           }
+  end
+
+  test "show" do
+    organizer = build(:organizer)
+    rendered_organizer = OrganizerJSON.show(%{organizer: organizer})
```

Comment:
In order to test the controller you can't do this, you must call the endpoints like in the other controller tests